src/app.rs

```diff
@@ -484,14 +485,26 @@ where
             info!("Rustc git commit: {}", hash);
         }
 
+        let thread_count :usize = env::var("AMETHYST_NUM_THREADS")
+            .as_ref()
+            .map(|s| s.as_str().parse()
+                    .expect("AMETHYST_NUM_THREADS was provided but is not a valid number!"))
+            .unwrap_or(0);
+
         let mut world = World::new();
 
         let thread_pool_builder = ThreadPoolBuilder::new();
         #[cfg(feature = "profiler")]
         let thread_pool_builder = thread_pool_builder.start_handler(|_index| {
             register_thread_with_profiler();
         });
-        let pool = thread_pool_builder.build().map(Arc::new)?;
+        let pool :ArcThreadPool;
+        if thread_count == 0 {
+            pool = thread_pool_builder.build().map(Arc::new)?;
+        } else {
+            info!("Running Amethyst with fixed thread pool: {}", thread_count);
```

Comment:
How about `debug` level? 

* the user specified the X thread count, echoing it back is okay at debug (when they want to check that it was read properly)
* hide information that a user doesn't need to see when they're not actively checking

Reply:
Fair enough, adjusting and fixing the merge conflict!